.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "shielder/PSP22"]
```

Comment:
how do we use it?

Reply:
had to be compiled before tests

Reply:
please open an issue in drink repository to support non-local contracts in tests and link the issue here

Reply:
https://github.com/inkdevhub/drink/issues/102